core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -41,6 +41,10 @@ package object config {
     .bytesConf(ByteUnit.MiB)
     .createWithDefaultString("1g")
 
+  private[spark] val DRIVER_MEMORY_OVERHEAD = ConfigBuilder("spark.driver.memoryOverhead")
```

Comment:
nit: should also add doc for this config here.

Reply:
Yes, it's added under `spark.driver.memory`. See https://github.com/apache-spark-on-k8s/spark/blob/0e8ca012dc6d5dfd5645f45b6d1bb84e1e9e72a5/docs/configuration.md.